sources/common/src/TimeUnit.cpp

```diff
@@ -0,0 +1,94 @@
+/// @file common/src/TimeUnit.cpp
+///
+/// The utilities.
+///
+/// The time unit library.
+///
+/// @par Full Description
+/// The functions for handling time units.
+///
+/// @par Copyright (c) 2017 vcdMaker team
+///
+/// Permission is hereby granted, free of charge, to any person obtaining a
+/// copy of this software and associated documentation files (the "Software"),
+/// to deal in the Software without restriction, including without limitation
+/// the rights to use, copy, modify, merge, publish, distribute, sublicense,
+/// and/or sell copies of the Software, and to permit persons to whom the
+/// Software is furnished to do so, subject to the following conditions:
+///
+/// The above copyright notice and this permission notice shall be included
+/// in all copies or substantial portions of the Software.
+///
+/// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+/// OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+/// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+/// THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+/// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+/// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+/// IN THE SOFTWARE.
+
+#include <algorithm>
+#include <stdexcept>
+#include <vector>
+#include <string>
+#include <ratio>
+
+#include "TimeUnit.h"
+#include "VcdException.h"
+
+const std::vector<std::string> TIME::Unit::TIME_UNITS =
+{ "s", "ms", "us", "ns", "ps", "fs" };
```

Comment:
Indent 4 spaces?

Reply:
Yep, I just reconfigured my editor to use spaces.